src/apps/backend/modules/account/internal/store/account-db.ts

```diff
@@ -15,22 +18,31 @@ export const AccountDbSchema: Schema = new Schema<AccountDB>(
       type: Boolean,
       required: true,
     },
+    contactNumber: {
+      default: null,
+      type: {
+        countryCode: String,
+        phoneNumber: String,
+      },
+      index: true,
+      unique: true,
```

Comment:
Is this true i.e. say system has two account with just email and password, and both have phone number as null. Would it throw?

Reply:
Yes right, we have to remove unique from the username too.